pkg/reconciler/common/testdata/kodata/additional-manifests/additional-resource.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: knative-additional
+  labels:
+    serving.knative.dev/release: "v0.16.1"
```

Comment:
Is this label actually taken into account while reading additional manifests.

If yes, sorry for missing it.

If not, having it there might be misleading, and give the impression that the operator evaluates it, when it doesn't.

Reply:
Yes.
If the user does not specify the field spec.version, we do not check if the version of the label matches the spec.version.
If the user specifies the field spec.version, we will validate if the version of the label matches the spec.version.